src/olympia/conf/stage/settings.py

```diff
@@ -103,6 +103,10 @@
     'z.pool': {'level': logging.ERROR},
 })
 
+# Update the logger name used for mozlog
+LOGGING['formatters']['json']['logger_name'] = 'http_app_addons_stage'
```

Comment:
I expect you did it deliberately but calling out this is a change - previously stage didn't override the name so used the same as prod used.

Reply:
it does set MOZLOG_NAME = "http_app_addons_stage", I simply forgot to remove it. I'll update the patch Monday morning

Reply:
ah

Reply:
Fixed in 3b77750793